sympy/physics/mechanics/body.py

```diff
@@ -0,0 +1,168 @@
+from sympy import Symbol
+from sympy.physics.mechanics import (RigidBody, Particle, ReferenceFrame,
+                                     inertia)
+from sympy.physics.vector import Point, Vector
+
+__all__ = ['Body']
+
+
+class Body(RigidBody, Particle):
+    """
+    A Body which can be connected by joints.
```

Comment:
I have not pushed updated code yet. Please give me a couple of minutes to do so. I was finishing other things as well.